cfn/configs/klaxon/dev/us-east-1/config.yml

```diff
@@ -412,10 +412,11 @@ context:
 
     environment:
       DATABASE_URL: "postgresql://{{resolve:secretsmanager:/klaxon/aurora-postgresql-dev/app:SecretString:username}}:{{resolve:secretsmanager:/klaxon/aurora-postgresql-dev/app:SecretString:password}}@{{resolve:secretsmanager:/klaxon/aurora-postgresql-dev/app:SecretString:writer-host}}:5432/klaxon"
-      ADMIN_EMAILS: "erik.reyna@washpost.com, armand.emamdjomeh@washpost.com, katlyn.alo@washpost.com"
```

Comment:
I added admins by putting the team's email addresses into my env file, connecting to the db and running the create admin command. Don't think we need to keep our email addresses stored here.